src/sass/elements/forms.scss

```diff
@@ -10,14 +10,14 @@ Provides classes for form elements
   font-size: 19px;
   font-weight: $mw-font-weight;
   line-height: $mw-line-height;
-  color: $mw-color-text-primary;
+  color: $mw-neutral-text;
   height: 44px;
   background: none;
   outline: none;
   box-shadow: none;
   margin: 0;
   &::placeholder {
-    color: $mw-color-inactive;
```

Comment:
Maybe inactive is a better name than placeholder?

Reply:
It used to be inactive but looking at that `&::placeholder` rule, it makes more sense to call it `placeholder` IMO.